rmgpy/molecule/group.py

```diff
@@ -1901,7 +1901,8 @@ def standardize_atomtype(self):
         modified = False
 
         # If this atom or any of its ligands has wild cards, then don't try to standardize
-        if self.hasWildCards: return modified
+        for atom in self.atoms:
```

Comment:
This might be better done in a separate cythonized function since it is an independent operation and the for loop would benefit from cythonization.

Reply:
@mjohnson541 Thanks for the comment! I added a Group level wildcards checker to loop through the atoms and check wildcards. This method is also added to group.pxd so it should be cythonized!!